### ISSUE TITLE
Custom installation directory example breaks uninstaller

### ISSUE THREAD

**Issue by gknapp** [electron-userland/electron-builder#5437]
* **Version**: 22.9.1

* **Electron Version**: 10.1.6
* **Electron Type (current, beta, nightly)**: current

* **Target**: Windows (10) using NSIS

I want to customise the suggested installation directory from `%PROGRAMFILES%\productName` to include a sub-directory `company\productName`. The docs detail [how to do this](https://www.electron.build/configuration/nsis#common-questions) with a NSIS script, however - when uninstalling my application, Windows reports it cannot find the device, path or file.

I've tried running the uninstall executable manually (also as admin in case it was permissions related). It's this executable that shows the error message box that the files cannot be found.

NSH script I'm using (most lifted from the docs):

```
!define INSTALLPATH "Company\App Name"
!macro preInit
  SetRegView 64
  WriteRegExpandStr HKLM "${INSTALL_REGISTRY_KEY}" InstallLocation "$PROGRAMFILES64\${INSTALLPATH}"
  WriteRegExpandStr HKCU "${INSTALL_REGISTRY_KEY}" InstallLocation "$PROGRAMFILES64\${INSTALLPATH}"
  SetRegView 32
  WriteRegExpandStr HKLM "${INSTALL_REGISTRY_KEY}" InstallLocation "$PROGRAMFILES\${INSTALLPATH}"
  WriteRegExpandStr HKCU "${INSTALL_REGISTRY_KEY}" InstallLocation "$PROGRAMFILES\${INSTALLPATH}"
!macroend
```

**Comment by gknapp** [comment]
I tried reverting back to a non-custom installer. I have removed the `"include"` field from my electron-builder configuration, so my script is not used.

I manually removed all traces of the GUID and app name in my registry. Re-installed and found the uninstaller still fails with the same error message.

![error-msg](https://user-images.githubusercontent.com/143027/100353078-e02d1800-2fe5-11eb-8abc-313a09c58394.png)

![registry](https://user-images.githubusercontent.com/143027/100353085-e4f1cc00-2fe5-11eb-931b-04d8174ed32c.png)
